swift_static_framework/swift_static_framework.bzl

```diff
@@ -85,13 +54,35 @@ def _swift_static_framework_impl(ctx):
         libraries.append(library)
 
         swift_info_provider = target[SwiftInfo]
-        swiftdoc = swift_info_provider.direct_swiftdocs[0]
-        swiftmodule = swift_info_provider.direct_swiftmodules[0]
-        swift_info_files += [swiftdoc, swiftmodule]
-        zip_args += [
-            _zip_swift_arg(module_name, swiftmodule_identifier, swiftdoc),
-            _zip_swift_arg(module_name, swiftmodule_identifier, swiftmodule),
-        ]
+        swiftdoc_input = swift_info_provider.direct_swiftdocs[0]
+        swiftmodule_input = swift_info_provider.direct_swiftmodules[0]
+
+        swiftdoc_output = None
+        swiftmodule_output = None
+
+        if swiftmodule_identifier == "arm64":
+            swiftdoc_output = ctx.outputs.arm64_swiftdoc
+            swiftmodule_output = ctx.outputs.arm64_swiftmodule
+        elif swiftmodule_identifier == "x86_64":
+            swiftdoc_output = ctx.outputs.x86_64_swiftdoc
+            swiftmodule_output = ctx.outputs.x86_64_swiftmodule
```

Comment:
Outside the scope of this change, but looks like we might be able to include the `swiftinterface` with next release of Swift Bazel rules: https://github.com/bazelbuild/rules_swift/commit/0cb4f29a2a8f3cb626cfb1da9c41427bc34d05f8

Just checking if we will only need to add an input/output for it or if it will require additional work?

Reply:
I think adding an input/output for it is enough to include `swiftinterface` files.